lib/pf/UnifiedApi/Controller/Auditlogs.pm

```diff
@@ -0,0 +1,52 @@
+package pf::UnifiedApi::Controller::Auditlogs;
```

Comment:
This should be RadiusAuditLogs since there may be other types of audit logs (admin audit log, etc)